### PR TITLE
Update docs to suggest JDK 17

### PR DESCRIPTION
**Pull request by JuliusZhou124**
#1609 
#1604

### CODE REVIEW

File: docs/source/docs/advanced-installation/sw_install/mac-os.md [hunk 1]
```diff
@@ -4,18 +4,14 @@
 Due to current [cscore](https://github.com/wpilibsuite/allwpilib/tree/main/cscore) restrictions, the PhotonVision server backend may have issues running macOS.
 :::
 
-:::{note}
-You do not need to install PhotonVision on a Windows PC in order to access the webdashboard (assuming you are using an external coprocessor like a Raspberry Pi).
```
Comment (mcm001) [inline docs/source/docs/advanced-installation/sw_install/mac-os.md:8]: We should keep this note but fix the os name

File: docs/source/docs/advanced-installation/sw_install/windows-pc.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ Bonjour provides more stable networking when using Windows PCs. Install [Bonjour
 
 ## Installing Java
 
-PhotonVision requires a JDK installed and on the system path. **JDK 11 is needed** (different versions will not work). You may already have this if you have installed WPILib, but ensure that running `java -version` shows JDK 11. If not, [download and install it from here](https://adoptium.net/temurin/releases?version=11) and ensure that the new JDK is being used.
+PhotonVision requires a JDK installed and on the system path. **JDK 17 is needed** (different versions will not work). *Windows Users must use the JDK that ships with WPILib.* Ensure that running `java -version` shows JDK 17. If not, [download and install it from here](https://github.com/wpilibsuite/allwpilib/releases/tag/v2025.1.1-beta-2) and ensure that the new JDK is being used.
```
Comment (mcm001) [inline docs/source/docs/advanced-installation/sw_install/windows-pc.md:15]: Do we want to mention -Dorg.gradle.java.home here too?

Comment (mcm001) [reply docs/source/docs/advanced-installation/sw_install/windows-pc.md:15]: Yeah. The two options you have are
- make sure the first/only Java on your PATH is the wpilib one
- specify the wpilib jdk with that option

Comment (JuliusZhou124) [reply docs/source/docs/advanced-installation/sw_install/windows-pc.md]: done

**Review by Juniormunk — APPROVED** [review]
lgtm

**Pull request merged**